docs/developer/quickstart.md

```diff
@@ -114,10 +114,29 @@ This will build the docker images `nginx-gateway-fabric:<your-user>` and `nginx-
 
 1. Create a `kind` cluster:
 
+   To create a `kind` cluster with IPv4 enabled:
+
    ```makefile
    make create-kind-cluster
    ```
 
+   To create a `kind` cluster with IPv6 or Dual IPFamily enabled, use this `config.yaml`:
+
+   ```yaml
+   kind: Cluster
+   apiVersion: kind.x-k8s.io/v1alpha4
+   nodes:
+   - role: control-plane
+   networking:
+     ipFamily: dual
+     # ipFamily: ipv6
+     apiServerAddress: 127.0.0.1
+   ```
+
+   ```shell
+   kind create cluster --config ~/cluster.yaml
```

Comment:
this command might create a different cluster than `make create-kind-cluster`. Because, we have the following in the Makefile:

```
create-kind-cluster: ## Create a kind cluster
        $(eval KIND_IMAGE=$(shell grep -m1 'FROM kindest/node' <$(SELF_DIR)tests/Dockerfile | awk -F'[ ]' '{print $$2}'))
        kind create cluster --image $(KIND_IMAGE)
```

Could we extend `create-kind-cluster:` target so that it uses the cluster.yaml with default (dual) configuration? For other configurations, we can ask the developer to modify that file here.